bin/setup

```diff
@@ -39,20 +39,6 @@ Dir.chdir APP_ROOT do
   puts "== Copying sample pwned passwords list =="
   run "cp pwned_passwords/pwned_passwords.txt.sample pwned_passwords/pwned_passwords.txt"
 
-  if ARGV.shift == "--docker" then
```

Comment:
Just removing all the docker stuff from the `setup` script, leaving it as-is otherwise.